tensorboard/backend/application.py

```diff
@@ -116,7 +116,9 @@ def _apply_tensor_size_guidance(sampling_hints):
     return tensor_size_guidance
 
 
-def standard_tensorboard_wsgi(flags, plugin_loaders, assets_zip_provider, experimental_plugins=[]):
+def standard_tensorboard_wsgi(
+    flags, plugin_loaders, assets_zip_provider, experimental_plugins=[]
```

Comment:
Use `experimental_plugins=None` instead and do the resolution inside the
function body, because:

  - default argument values are evaluated at function definition time,
    not invocation time, so this introduces a globally shared mutable
    list object—not what you want! (cf. [style guide][style])
  - using `None` as a default argument value in general is a [common
    Python pattern][none] enabling callers to specify “use default”
    without having to duplicate the actual value of the default

(Likewise below.)

[style]: https://google.github.io/styleguide/pyguide.html#2124-decision
[none]: https://unhashable.com/always-use-none-for-default-args-in-python/